net/openthread-br/Makefile

```diff
@@ -48,31 +48,22 @@ define Package/openthread-br/description
   A Thread border router for POSIX-based platforms.
 endef
 
-define Package/openthread-br/conffiles
-/var/lib/thread
-endef
-
 CMAKE_OPTIONS += \
-	-DOT_BORDER_ROUTER:BOOL=ON \
-	-DOT_BORDER_ROUTING_NAT64:BOOL=ON \
-	-DOT_CHANNEL_MANAGER:BOOL=ON \
-	-DOT_CHANNEL_MONITOR:BOOL=ON \
-	-DOT_COMMISSIONER:BOOL=ON \
-	-DOT_ECDSA:BOOL=ON \
-	-DOT_FIREWALL:BOOL=OFF \
-	-DOT_SERVICE:BOOL=ON \
-	-DOT_SRP_CLIENT:BOOL=ON \
-	-DOT_SRP_SERVER:BOOL=ON \
-	-DOTBR_BACKBONE_ROUTER:BOOL=ON \
-	-DOTBR_BORDER_ROUTING:BOOL=ON \
-	-DOTBR_DNSSD_DISCOVERY_PROXY:BOOL=ON \
-	-DOTBR_DUA_ROUTING:BOOL=ON \
+	-DOT_BORDER_ROUTER=ON \
+	-DOT_CHANNEL_MANAGER=ON \
+	-DOT_CHANNEL_MONITOR=ON \
+	-DOT_FIREWALL=OFF \
+	-DOTBR_BACKBONE_ROUTER=ON \
+	-DOTBR_BORDER_ROUTING=ON \
+	-DOTBR_DNSSD_DISCOVERY_PROXY=ON \
+	-DOTBR_DUA_ROUTING=ON \
 	-DOTBR_MDNS=mDNSResponder \
-	-DOTBR_OPENWRT:BOOL=ON \
-	-DOTBR_REST:BOOL=ON \
-	-DOTBR_SRP_ADVERTISING_PROXY:BOOL=ON \
-	-DOTBR_SRP_SERVER_AUTO_ENABLE:BOOL=ON \
-	-DOTBR_TREL:BOOL=ON
+	-DOTBR_NAT64=ON \
```

Comment:
This does not have the same effect as enabling OT_BORDER_ROUTING_NAT64. At the end of the build, it shows the enabled OPENTHREAD_CONFIG_ options:

Current:
OPENTHREAD_CONFIG_NAT64_BORDER_ROUTING_ENABLE=1
OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE=1


Vs after applying your PR:
OPENTHREAD_CONFIG_NAT64_BORDER_ROUTING_ENABLE=0                                                                                                           OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE=0

Reply:
Interesting. Ran clean/build again and now it does not show this behaviour.

Reply:
I've just re-checked it on a fully clean build - both options are `1`. OT_BORDER_ROUTING_NAT64 is not even exists anywhere.